.pipelines/multitenancy/swiftv2-e2e-step-template.yaml

```diff
@@ -26,51 +30,57 @@ steps:
-        envsubst '$NODE_NAME_0' < test/integration/manifests/swiftv2/mtpod0.yaml | kubectl apply -f -
-        export NODE_NAME_1="$(kubectl get nodes -o json | jq -r .items[1].metadata.name)"
-        echo $NODE_NAME_1
-        echo "Start another pod using the reserved IP"
-        envsubst '$NODE_NAME_1' < test/integration/manifests/swiftv2/mtpod1.yaml | kubectl apply -f -
-        sleep 2m
-        kubectl get pod -o wide -A
-        sleep 2m
-        echo "Check pods after 4 minutes"
-        kubectl get po -owide -A
-        kubectl describe pni
-    name: "start_swiftv2_pods"
-    displayName: "Start Swiftv2 Pods"
-    continueOnError: ${{ parameters.continueOnError }}
-    env:
-      SUBNET_TOKEN: $(SUBNET_TOKEN)
+        set -ex
+        source ./.pipelines/multitenancy/scripts/utils.sh
+        export_envVars ${{ parameters.scenario }} 
+        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }} GROUP=$RG
+    name: "kubeconfig"
+    displayName: "Set Kubeconfig"
 
-  - script: |
-      set -e
-      kubectl get po -owide -A
-      cd test/integration/swiftv2
-      echo "TestSwiftv2PodToPod and will run it after migration from scripts."
-      go test -count=1 swiftv2_test.go -timeout 3m -tags swiftv2 -run ^TestSwiftv2PodToPod$ -tags=swiftv2,integration -v
-    retryCountOnTaskFailure: 3
-    name: "Swiftv2_Tests_future_version"
-    displayName: "Swiftv2 Tests through code"
-    continueOnError: ${{ parameters.continueOnError }}
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(ACN_TEST_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        set -ex
+        source ./.pipelines/multitenancy/scripts/utils.sh
+        export_envVars ${{ parameters.scenario }} 
+        echo "Deploying on Linux nodes"
+        export CNI_IMAGE=$(make cni-image-name-and-tag-multiarch CNI_VERSION=$(make cni-version))
+        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }} GROUP=$RG
+        echo "Keep CNS version up to date, grabbing pipeline parameter"
+        CNS_IMAGE=$(make cns-image-name-and-tag-multiarch)
+        sed -i '/containers:/{n;n;s/\(image\).*/\1: '"${CNS_IMAGE//\//\\/}"'/}' ./test/integration/manifests/cns/daemonset-linux.yaml
+        sed -i '/initContainers:/{n;n;s/\(image\).*/\1: '"${CNI_IMAGE//\//\\/}"'/}' ./test/integration/manifests/cns/daemonset-linux.yaml
+        sed -i '/- name: cni-installer/,/volumeMounts:/ {    
+          /command: \["\/dropgz"\]/!b; a \ \ \ \ \ \ \ \ \ \ args:\n\ \ \ \ \ \ \ \ \ \ \ \ - deploy\n\ \ \ \ \ \ \ \ \ \ \ \ - --skip-verify\n\ \ \ \ \ \ \ \ \ \ \ \ - azure-vnet\n\ \ \ \ \ \ \ \ \ \ \ \ - -o\n\ \ \ \ \ \ \ \ \ \ \ \ - /opt/cni/bin/azure-vnet\n\ \ \ \ \ \ \ \ \ \ \ \ - azure-vnet-telemetry\n\ \ \ \ \ \ \ \ \ \ \ \ - -o\n\ \ \ \ \ \ \ \ \ \ \ \ - /opt/cni/bin/azure-vnet-telemetry
```

Comment:
I thought dropgz is deprecated already from the repo?

Reply:
oh, this is just referring to the command string in the daemonset-linux file for matching purposes (this command is just making image tag replacements for in the azure-cns daemonset yaml thats already being used). The image we are using is azure-cni.

Reply:
we apply the daemonset in the next couple of lines, but first we have to update the yaml first with the image and args we want so that the cni image gets deployed.